build_tools/circle/build_test_pypy.sh

```diff
@@ -21,15 +21,23 @@ which python
 # XXX: numpy version pinning can be reverted once PyPy
 #      compatibility is resolved for numpy v1.6.x. For instance,
 #      when PyPy3 >6.0 is released (see numpy/numpy#12740)
-pip install --extra-index https://antocuni.github.io/pypy-wheels/ubuntu "numpy==1.15.*" Cython pytest
-pip install "scipy>=1.1.0" sphinx numpydoc docutils joblib pillow
+pip install --extra-index https://antocuni.github.io/pypy-wheels/ubuntu numpy Cython pytest
+pip install scipy sphinx numpydoc docutils joblib pillow
 
 ccache -M 512M
 export CCACHE_COMPRESS=1
 export PATH=/usr/lib/ccache:$PATH
 export LOKY_MAX_CPU_COUNT="2"
+export OMP_NUM_THREADS="1"
 
-pip install -vv -e . 
+pip install -e .
```

Comment:
verbose can be good to see cython build errors?

Reply:
Yeah, but wouldn't it print the traceback on errors anyway? Or is that less detailed then?

The problem of verbose if that it makes CircleCI logs harder to parse since it's done in the same step as tests.